x-pack/plugins/ml/public/application/components/job_spaces_selector/cannot_edit_callout.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { EuiSpacer, EuiCallOut } from '@elastic/eui';
+
+export const CannotEditCallout: FC<{ jobId: string }> = ({ jobId }) => (
+  <>
+    <EuiCallOut
+      color="warning"
+      iconType="help"
+      title={i18n.translate('xpack.ml.management.spacesSelectorFlyout.cannotEditCallout.title', {
+        defaultMessage: 'Insufficient permissions to edit spaces for {jobId}',
+        values: { jobId },
+      })}
+    >
+      <FormattedMessage
+        id="xpack.ml.management.spacesSelectorFlyout.cannotEditCallout.text"
+        defaultMessage="To change this job's spaces, you need privileges to modify jobs in all spaces. Contact your system administrator for more information."
```

Comment:
Tried to make this clearer, but it's still unclear to me whether you need authority to modify *all* jobs in all spaces or just *this* job in all spaces...and which privileges give you that authority.
```suggestion
        defaultMessage="To change the spaces for this job, you need authority to modify jobs in all spaces. Contact your system administrator for more information."
```

Reply:
https://github.com/elastic/kibana/pull/83320/commits/e90c0917cf0e1297beafdebc3ad7488c7ada46a6